packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -423,28 +550,27 @@ abstract class ImageStreamCompleter with Diagnosticable {
   /// This callback will never fire if [removeListener] is never called.
   void addOnLastListenerRemovedCallback(VoidCallback callback) {
     assert(callback != null);
+    _checkDisposed();
```

Comment:
Update this method's docs to make it clear that this is about active listeners and passive ones don't count?

Reply:
Added a doc.